### PR TITLE
Minicart Image

### PR DESCRIPTION
**Pull request by emersonlaurentino**
## What's the purpose of this pull request?
Change the size of minicart image

## How it works? 
I created a component to image on the minicart

## How to test it?
open the minicart

## References
<em>Spread the knowledge: is there any content you used to create this PR that is worth sharing?</em>  

<em>Extra tip: adding references to related issues or mentioning people important to this PR may be good for the documentation and reviewing process</em>

### CODE REVIEW

File: packages/gatsby-theme-vtex/src/components/Minicart/Drawer/Image.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  ...props
+}) => {
+  const imageProps = {
+    loading,
+    ...props,
+  }
+
+  return (
+    <Box variant={props.variant}>
+      <Box {...imageProps} />
```
Comment (tlgimenes) [inline packages/gatsby-theme-vtex/src/components/Minicart/Drawer/Image.tsx:25]: Do this box has the same variant from the box above ?

Comment (tlgimenes) [inline packages/gatsby-theme-vtex/src/components/Minicart/Drawer/Image.tsx:25]: This way we create one object less
```suggestion
      <Box loading={loading} {...props} />
```

Comment (emersonlaurentino) [reply packages/gatsby-theme-vtex/src/components/Minicart/Drawer/Image.tsx:25]: this is a mistake, I will fix it!

Comment (emersonlaurentino) [reply packages/gatsby-theme-vtex/src/components/Minicart/Drawer/Image.tsx:25]: loading attribute does not exist on the box component, with object he infer the props. Do you have an idea how we fix it?

Comment (emersonlaurentino) [reply packages/gatsby-theme-vtex/src/components/Minicart/Drawer/Image.tsx:26]: Screen Shot 2020-11-19 at 12 22 33

Comment (tlgimenes) [reply packages/gatsby-theme-vtex/src/components/Minicart/Drawer/Image.tsx:26]: Oops, ok

**Review by tlgimenes — APPROVED** [review]

**Pull request merged**